src/tutorials/0004-mutable-file-system/02.js

```diff
@@ -8,7 +8,12 @@ const validate = async (result, ipfs) => {
   } else if (!!result && result.cid.toString() === 'QmUNLLsPACCz1vLxQVkXqqLX5R1X345qqfHbsf67hvA3Nn') {
     return {
       success: utils.validationMessages.SUCCESS,
-      logDesc: "Here's the status of your root directory ( `/` ). Notice that it has a CID even though it doesn't have contents yet. Every empty IPFS node has this exact same CID, because their non-existent contents are identical!",
+      logDesc: [
+        "Here's the status of your root directory (`/`).",
+        `<pre class="code-highlight"><code class="hljs json">${JSON.stringify(result, null, 2)}</code></pre>`,
+        'The output is very long because the CID is represented as an `Object` internally. To simplify, this we can use the `toString()` method on the `cid` property to get the CID in string format: `' + result.cid.toString() + '`. In future lessons we\'ll always show this simplified version to make it easier to read as shown bellow. <br/> <br/>',
```

Comment:
```suggestion
        'The output is very long because the CID is represented as an `Object` internally. To simplify, this we can use the `toString()` method on the `cid` property to get the CID in string format: `' + result.cid.toString() + '`. In future lessons we\'ll always show this simplified version to make it easier to read, as shown below. <br/> <br/>',
```